foundation/foundation-mda/src/main/resources/templates/general-docker/training.docker.pom.xml.vm

```diff
@@ -87,6 +87,9 @@
             <plugin>
                 <groupId>${group.fabric8.plugin}</groupId>
                 <artifactId>docker-maven-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
+                </configuration>
```

Comment:
I: temporary skip the training pipeline docker build, this will be removed by the followup ticket

Reply:
Haha I left my question before reading all the way through. Whoops!